src/main/java/com/github/sergdelft/j2graph/walker/json/JsonVisitor.java

```diff
@@ -0,0 +1,255 @@
+package com.github.sergdelft.j2graph.walker.json;
+
+import com.github.sergdelft.j2graph.graph.NonTerminal;
+import com.github.sergdelft.j2graph.graph.Symbol;
+import com.github.sergdelft.j2graph.graph.Token;
+import com.github.sergdelft.j2graph.graph.Vocabulary;
+import com.github.sergdelft.j2graph.walker.Walker;
+import com.google.gson.*;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+
+/**
+ * Visitor to help generate data for ICLR20-Great data format: https://github.com/VHellendoorn/ICLR20-Great
+ * <p>
+ * The Visitor also mutates binary expressions (<,<=,>,>=)
+ */
+public class JsonVisitor implements Walker {
```

Comment:
Maybe we should call this class `OffSideVisitor`, so that we fully couple it to your work? Otherwise, this might confuse others.

Reply:
Makes sense, I was thinking the same thing about the JsonVisitor being ambiguous actually.

Reply:
@mauricioaniche , perhaps ICLR20GreatVisitor is better?

Reply:
Done, named to ICLR20GreatVisitor and also named the DataGenerator to ICLR20GreatDataGenerator.